src/ClientNet462/Program.cs

```diff
@@ -0,0 +1,148 @@
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Client
+{
+    internal class Program
+    {
+        static async Task Main(string[] args)
+        {
+            try
+            {
+                using (var cts = new CancellationTokenSource())
+                {
+                    ConsoleCancelEventHandler handler = (o, e) =>
+                    {
+                        Console.WriteLine("Cancelling...");
+                        cts.CancelAfter(0);
+                        e.Cancel = true;
+                    };
+                    Console.CancelKeyPress += handler;
+                    try
+                    {
+                        Console.WriteLine("Searching for city 'detroit' (press Ctrl-C to cancel)...");
+                        var result = await CallGraphQLAsync<MyResponseData>(
+                            new Uri("https://graphql-weather-api.herokuapp.com/"),
+                            HttpMethod.Post,
+                            "query ($city: String!) { getCityByName(name: $city) { name country } }",
+                            new
+                            {
+                                city = "detroit",
+                            },
+                            cts.Token);
+                        if (result.Errors?.Count > 0)
+                        {
+                            Console.WriteLine($"GraphQL returned errors:\n{string.Join("\n", result.Errors.Select(x => $"  - {x.Message}"))}");
+                        }
```

Comment:
Figured something basic to show returned errors wouldn't hurt.

Reply:
👍